test/parallel/test-child-process-spawn-error.js

```diff
@@ -53,3 +53,31 @@ enoentChild.on('error', common.mustCall(function(err) {
   assert.strictEqual(err.path, enoentPath);
   assert.deepStrictEqual(err.spawnargs, spawnargs);
 }));
+
+
+// Verify that the spawn() function throws when the file parameter is not a valid string
+{
+  assert.throws(() => {
+    spawn(123, common.mustNotCall());
+  }, {
+    code: 'ERR_INVALID_ARG_TYPE',
+    name: 'TypeError',
+    message: 'The "file" argument must be of type string. Received type number (123)'
+  });
```

Comment:
Can we do the same in this file?
```suggestion
  }, { code: 'ERR_INVALID_ARG_TYPE' });
```